ui/components/app/wallet-overview/eth-overview.js

```diff
@@ -254,6 +259,8 @@ const EthOverview = ({ className }) => {
                   properties: {
                     location: 'Home',
                     text: 'Bridge',
+                    chain_id: chainId,
```

Comment:
Just a note: chain_id is currently being affixed to every event payload in the metametrics controller level. I *assume* this will have to change for multi-network, but i'm not sure how it will be handled at this point. If we do have to manually attach it at the UI level thats fine, but until we remove the auto-affix this is going to get overwritten